rpi5.yaml

```diff
@@ -299,6 +299,15 @@ images:
           "dom0/z_blinky.bin": "%{ZEPHYR_DOMU2_DIR}/%{ZEPHYR_DOMU2_BUILD_DIR}/zephyr/zephyr.bin"
           "dom0/helloworld_xen-arm64": "domu_unikraft/helloworld_xen-arm64"
```

Comment:
Those have to be removed then if Dom0 is separate now and mandatory.

Reply:
Removed